generator/src/main/java/com/scottlogic/deg/generator/restrictions/NumericRestrictions.java

```diff
@@ -3,15 +3,34 @@
 import java.math.BigDecimal;
 import java.util.Objects;
 
+import static com.scottlogic.deg.generator.utils.NumberUtils.coerceToBigDecimal;
+
 public class NumericRestrictions {
+    public static final int DEFAULT_NUMERIC_SCALE = 20;
+    private final int numericScale;
+    public NumericLimit<BigDecimal> min;
```

Comment:
Possibly out-of-scope comment, but it would be great if these could be final and then this type immutable.